src/core/routers/v1/cores.ts

```diff
@@ -0,0 +1,15 @@
+import { Router } from 'express';
+import { FactoryFunction } from 'tsyringe';
+import { CoresController } from '../../controllers/cores';
+
+const coresRoutesFactory: FactoryFunction<Router> = (dependencyContainer): Router => {
+  const coresRouter = Router();
+  const controller = dependencyContainer.resolve(CoresController);
+
+  coresRouter.route('/cores').get(controller.getCores).post(controller.createCore);
+  coresRouter.route('/cores/:coreId').get(controller.getCoreByID);
```

Comment:
```suggestion
  coresRouter.get('/', controller.getCores);
  coresRouter.post('/', controller.getCores);
  coresRouter.get('/:coreId', controller.getCoreByID);
```
then in global.ts
```ts
import { Router } from 'express';
import { container } from 'tsyringe';
import { coresRoutesFactory } from '../../core/routers/v1/cores';
import { openapiRouterFactory } from './openapi';

const globalRoutesFactory = (): Router => {
  const globalRouter = Router();
  globalRouter.use(openapiRouterFactory(container));
  globalRouter.use('/api/v1/cores', coresRoutesFactory(container));
  return globalRouter;
};
export { globalRoutesFactory };

```


even though it behaves the same, all of our services are implemented like the suggestion, let's stick to it.

Reply:
done